### PR TITLE
perf: optimize flags icons

### PR DESCRIPTION
**Pull request by ivan-kalachikov**
### Jira-link:
https://virtocommerce.atlassian.net/browse/VCST-1214
### Artifact URL:
https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.58.0-pr-1087-17dc-17dc83cf.zip

### CODE REVIEW

File: client-app/shared/layout/components/language-selector/language-selector.vue [hunk 1]
```diff
@@ -11,9 +11,12 @@
       ></span>
 
       <span
-        class="fi fis fa-2x -my-3 h-[30px] !w-[30px] rounded-full lg:h-[14px] lg:!w-[14px]"
-        :class="`fi-${currentLanguage?.twoLetterRegionName.toLowerCase()}`"
-      ></span>
+        class="-my-3 flex h-[30px] !w-[30px] items-center justify-center overflow-hidden rounded-full lg:h-[14px] lg:!w-[14px]"
+      >
+        <span class="country-flag text-[40px] lg:text-[18px]">{{
```
Comment (goldenmaya) [inline client-app/shared/layout/components/language-selector/language-selector.vue:16]: text-6xl lg:text-[1.75rem]

File: client-app/shared/layout/components/language-selector/language-selector.vue [hunk 2]
```diff
@@ -98,4 +104,16 @@ function select(locale: string) {
...
 }
+
+function getFlagEmoji(countryCode: string): string {
+  // commonly accepted approach to convert country code to flag emoji
+  return countryCode.toUpperCase().replace(/./g, (char) => String.fromCodePoint(char.charCodeAt(0) + 127397));
+}
 </script>
+
+<style scoped>
+.country-flag {
```
Comment (goldenmaya) [inline client-app/shared/layout/components/language-selector/language-selector.vue:115]: Specify font-flag class in tailwing.config.ts and use it in template

https://tailwindcss.com/docs/font-family#customizing-your-theme

**Review by goldenmaya — CHANGES_REQUESTED** [review]

File: client-app/shared/layout/components/language-selector/language-selector.vue (hunk 1 above)
Comment (ivan-kalachikov) [reply client-app/shared/layout/components/language-selector/language-selector.vue]: they are a bit different numbers in your example, but I got the idea and changed values

**Review by goldenmaya — CHANGES_REQUESTED** [review]
Actual result:
Screenshot 2024-05-14 at 12 45 53
Screenshot 2024-05-14 at 12 46 04

Expected result:
Screenshot 2024-05-14 at 12 43 54
Screenshot 2024-05-14 at 12 44 05

**Pull request closed without merging**